ecpy_hqc_legacy/instruments/drivers/visa/cryomagnetics_cs4.py

```diff
@@ -197,8 +213,13 @@ def activity(self):
     @secure_communication()
     def activity(self, value):
         par = _ACTIVITY_DICT.get(value, None)
+        if par != 'PAUSE':
```

Comment:
I don't think 'SWEEP ZERO FAST' and 'SWEEP ZERO SLOW' work; we'll see.

Reply:
maybe we can just remove the 'ZERO' from the activity dict since you don't use it.

Reply:
Removing zero should be fine (it was not there before).